### ISSUE TITLE
Feature Request: Mark Items as read

### ISSUE THREAD

**Issue by Yakito** [feedbin/feedbin#196]
It would be good to be able to mark items as read as I scroll through them. Sometimes I have lots of feeds to read and I want the ones I already saw to be marked as read while not having to mark the whole feed as read.
I would also have the option to disable the confirmation asking if I am sure I want to mark all items as read.
Also, even though there is a nice keyboard shortcut for adding a star, it will also be fast to add a little star next to the feed to be able to set it.

**Comment by alexkrolick** [comment]
I have the same request. It would be even cooler if it could be triggered with a swipe gesture.

**Comment by screendriver** [comment]
Any news on this? The issue was opened 2 years ago and other feed readers like _Tiny Tiny RSS_ support that very good :confused:

**Comment by tonydehnke** [comment]
This is a feature I'd really like to see as well.  It's been one of the best features I've liked from Bazquk, FeedlerPro and others..  A bit surprised not to see it in this app.

**Comment by rohitsathish** [comment]
Yes, I agree. Marking items as read once they've been scrolled past is a critical feature present in nearly all other competitors. Disappointing to see that this hasn't been implemented in 6 years. A major feature missing in an otherwise great product.
